src/Nri/Ui/CssFlexBoxWithVendorPrefix.elm

```diff
@@ -0,0 +1,292 @@
+module Nri.Ui.CssFlexBoxWithVendorPrefix
```

Comment:
I think it would be better to publish this as it's own module. This could  be useful to others as well. and maybe name it `Css.FlexBox`.

Reply:
Happy to split this out! One question: What's your motivation to drop `...WithVendorPrefix` from the name? To me `Css.FlexBox` gives the impression elm-css itself doesn't when really it does, just not with prefixes.

Reply:
cc @stoeffel